content/html/concepts/tags/terms/font/font.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: '<font>'
+Description: 'Used to be used to set the font of text. This is now deprecated.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Style'
+  - 'Typography'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+  
+The `<font>` tag was used to set the font of text. It is not used any more in favor of the [CSS](https://www.codecademy.com/resources/docs/css) [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property instead because the `<font>` tag is now deprecated.
```

Comment:
```suggestion
The **`<font>`** tag was used to set the font of text. It is not used any more in favor of the [CSS](https://www.codecademy.com/resources/docs/css) [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property instead because the `<font>` tag is now deprecated.
```